docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -126,7 +114,9 @@ Standards for developing module and plugin utilities
 ====================================================
 
 * ``module_utils`` and ``plugin_utils`` can be marked for only internal use in the collection, but they MUST document this and MUST use a leading underscore for file names.
-* It is a breaking change when you make an existing ``module_utils`` private and in that case the collection requires a major version bump.
+
+  * It is a breaking change when you make an existing ``module_utils`` private and in that case the collection requires a major version bump.
```

Comment:
```suggestion
  * It is a breaking change when you make an existing ``module_utils`` private, in which case the collection requires a major version bump.
```

Reply:
Should this be formatted as a note rather than an indented list item?

Reply:
If i understand it correctly, we agreed to put "main" statements at the top level and all related notes will go at the next level with respect to them.